doc/spec.md

```diff
@@ -2846,12 +2852,12 @@ application-specific dialect) without breaking existing programs.
 
 ### any
 
-`any(x)` returns `True` if any element of the iterable sequence x is true.
+`any(x)` returns `True` if any element of the iterable sequence x has a truth value of true.
 If the iterable is empty, it returns `False`.
 
 ### all
 
-`all(x)` returns `False` if any element of the iterable sequence x is false.
+`all(x)` returns `False` if any element of the iterable sequence x has a truth-value of false.
```

Comment:
"truth value" above, "truth-value" here. Either is fine, but pick one. :)

Reply:
Done, in https://github.com/google/starlark-go/pull/142.